Ideas/easier-embedded-rust.md

```diff
@@ -0,0 +1,57 @@
+# An Easier Embedded Rust
+
+## Problem Statement
+
+**How could Rust make it easier for people just getting started with Rust and/or Embedded?**
+
+Rust has made embedded development better (for me) in a number of ways:
+
+* Leveraging what Rust already gives you:
+    * Tools like Cargo for build + package management
+    * Ability to run `no_std` programs easily
+    * Modern language features (modules, iterators, type system)
+    * Making it "harder to get things wrong" - use the compiler to detect errors statically
+* Building good tools:
+    * Way better debugger tools via probe-rs, and tools built on top like probe-run and probe-rs-cli
+    * Tooling for logging, like defmt
+* Reasonably good docs, and teaching materials
+
+However, Rust does not have as good of a "getting started (new to embedded and/or new to rust" story, or a story for "just want things to work (artists/hobbyists)".
+
+For whatever shortcomings (performance, reliability, licensing, "under the hood complexity") exist (or are perceived to exist) for ecosystems like Arduino and CircuitPython/MicroPython: they make getting started much easier than Rust does today.
+
+## Concepts to consider
+
+Balancing "simple" and "easy". How can we make things *easier*, without hiding all of the complexity under the hood, or making things more complex just so we can hide them under the hood?
+
+Hidden complexity makes it very hard if you ever fall off the "happy path". Requiring "extra" tools not typically used by "production" embedded rust users makes it harder to find help. Special systems means effort is split.
```

Comment:
I think a couple more words here would help with clarity
- harder to find help (in the community? with the tools?)
- Special systems (with these extra, hard to acquire tools?) means effort is split (how?)